site/content/docs/5.1/components/orange-local-headers.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: docs
+title: Orange local headers
+description: Documentation and examples for Boosted's exclusive Brand responsive title bars, breadcrumbs and local navigation bars.
+group: components
+toc: true
+---
+
+## Examples
+
+Below are some examples. For more, please refer to our [examples page]({{<docsref "/examples/title-bars">}}).
+
+### Title bars
+
+#### On black
+
+{{< example class="p-0" >}}
+<div class="bg-dark title-bar">
+  <div class="container">
+    <h1>Title</h1>
+  </div>
+</div>
+{{</ example >}}
+
+#### With image
+
+{{< example class="p-0">}}
+<div class="bg-supporting-pink title-bar">
+  <div class="container">
+    <h1>Title</h1>
+    <picture>
+      <source media="(min-width:1440px)" srcset="/docs/{{< param docs_version >}}/assets/img/title-bars-illustrations/illustration-1440.png">
+      <source media="(min-width:1280px)" srcset="/docs/{{< param docs_version >}}/assets/img/title-bars-illustrations/illustration-1280.png">
+      <source media="(min-width:1024px)" srcset="/docs/{{< param docs_version >}}/assets/img/title-bars-illustrations/illustration-1024.png">
+      <source media="(min-width:768px)" srcset="/docs/{{< param docs_version >}}/assets/img/title-bars-illustrations/illustration-768.png">
+      <source media="(min-width:480px)" srcset="/docs/{{< param docs_version >}}/assets/img/title-bars-illustrations/illustration-480.png">
+      <source media="(min-width:320px)" srcset="/docs/{{< param docs_version >}}/assets/img/title-bars-illustrations/illustration-320.png">
+      <img src="/docs/{{< param docs_version >}}/assets/img/title-bars-illustrations/illustration-320.png" alt=""/>
+    </picture>
+  </div>
+</div>
+{{</ example >}}
+
+### Breadcrumb
+
+{{< callout info >}}
+Example will come with issue [#891]({{<param repo>}}/issues/891).
+{{</ callout >}}
+
+### Local navigation bar
+
+{{< callout info >}}
+Example will come with issue [#890]({{<param repo>}}/issues/890).
+{{</ callout >}}
+
+## Title bars
+
+A title bar is designed to display the title of a page and shall be placed right below a global header.
+
+Being based on spacing and backgrounds utilities, title bars were designed to take `100vw` width.
+
+### Background color
+
+To change the appearance of a title bar, use our [background utilities]({{<docsref "/utilities/background">}}). Orange branded background utilities are `.bg-dark`, `.bg-white` and `.bg-supporting-*`.
```

Comment:
Are we sure we can use all supporting colors? It is not specified in the [Design Guidelines](https://system.design.orange.com/0c1af118d/p/774477-local-headers/b/743cd0) where we can only see white/black/green background.

Reply:
To discuss with designers, I'll contact them about that